modules/osdk-managing-psa-for-operators-with-escalated-permissions.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-complying-with-psa.adoc
+
+:_content-type: PROCEDURE
+[id="osdk-managing-psa-for-operators-with-escalated-permissions.adoc_{context}"]
+= Managing pod security admission for Operator workloads that require escalated permissions
+
+If your Operator project requires escalated permissions to run, you cannot rely on pod security admission synchronization to label your Operator's namespaces. You must set your Operator's namespace label with the appropriate pod security level.
+
+[IMPORTANT]
+====
+If you do not set your Operator's namespace label, you must include namespace labelling as part of your Operator's manual installation instructions.
+====
+
+.Procedure
+
+. Set the security context constraint (SCC) to the required permission level as shown in the following example:
```

Comment:
```suggestion
. Set the security context configuration to the required permission level as shown in the following example:
```

SCC is indeed security context constraints, but that's the rules for the cluster.  Pods/containers configure what we've been calling their "security context configuration" (the configuration for their security context, which will then be compared against the available security context constraints, to see if the pod can be allowed, or under which SCC it is allowed)

Reply:
Addressed in dee0dc7